apisix/core/ctx.lua

```diff
@@ -96,6 +96,9 @@ do
             elseif key == "service_id" then
                 val = ngx.ctx.api_ctx and ngx.ctx.api_ctx.service_id
 
+            elseif key == "consumer_name" then
```

Comment:
The same value can also be obtained when using `consumer_id`.

Reply:
ok, I will add it later.